libs/labelbox/src/labelbox/schema/label_score.py

```diff
@@ -0,0 +1,19 @@
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class LabelScore(DbObject):
+    """
+    a label score
+
+    Attributes
+    name
+    score
+
+    """
```

Comment:
would be nice to preserve the same docs style format as with the rest of schema objects. Look `Dataset`, `Batch` for examples.

Reply:
If I understand this class correctly, this is just a `data` class.
My personal opinion you should use a pydantic model instead of `DbObject` as a parent here. DbObject is our home-grown graphql client framework. Unless your class executes queries using this framework, no need to use it